lib/airnow_api_adapter.rb

```diff
@@ -0,0 +1,75 @@
+class AirnowApiAdapter
+  include HTTParty
+  base_uri 'http://www.airnowapi.org'
+
+  def query_params
+    {
+      'API_KEY': Rails.application.secrets.air_now_api_key,
+      format: "application/json"
+    }
+  end
+
+  def readings_for(zip_code)
+    self.class.get("/aq/observation/zipCode/current", {
+      query: query_params.merge({
+        distance: '25',
+        zipCode: zip_code
+      })
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.